docs/troubleshooting-guide/sql-logic.md

```diff
@@ -0,0 +1,261 @@
+
+**Explanation**
+
+Rows can get removed by accident when data gets refreshed in upstream systems or ETL jobs. 
+
+The order of the tables used in your `JOIN` clause matters. For example:
+
+- When you write a `LEFT JOIN`, the table that comes *before* the `JOIN` clause in your query is “on the left".
+- The rows from the table “on the right” will be filtered out if they don’t meet your join condition in the `ON` clause.
+
+The execution order of the query may combine your join conditions and WHERE clauses in ways that you might not expect.
+
+**Further reading**
+
+- [Common problems with SQL joins][common-join-problems] 
+
+#### My aggregations (counts, sums, etc.) are too high.
+
+1. Check if your source tables or upstream queries have [duplicated rows](#my-result-has-duplicated-rows).
+2. Check your source tables or upstream queries for rows that should be filtered out.
+    - Are you including empty or `NULL` rows in your aggregations?
+    - Ask your Metabase admin or data team about business logic that defines invalid, cancelled, or expired records that should be filtered out in queries.
+3. If you’re aggregating unique values, check that you’re not double-counting them.
+    - Do you need to use `COUNT_DISTINCT` instead of `COUNT`?
+    - Are you applying a `SUM` on top of a `COUNT_DISTINCT`?
+4. If you’re getting an error message, go to [Troubleshooting error messages][troubleshooting-error-messages].
+5. If you’re still stuck, search or ask the [Metabase community][discourse].
+
+#### My aggregations (counts, sums, etc.) are too low.
+
+1. Check if your source tables or upstream queries have [missing rows](#my-result-has-missing-rows).
+2. If you’re getting an error message, go to [Troubleshooting error messages][troubleshooting-error-messages].
+3. If you’re still stuck, search or ask the [Metabase community][discourse].
+
+### I don’t know if I’m using a nested query.
+
+You have a nested query if your SQL contains:
+
+- **More than one `SELECT` statement**
+
+    You’re using subqueries.
+    
+- **A `WITH` clause**
+    
+    You’re using CTEs (Common Table Expressions).
+    
+- **Notation that looks like `{{ variable_name }}`** in your `FROM` or `WITH` clause.
+    
+    You have a Metabase SQL variable that references a saved question or model.
+    
+**Further reading**
+
+- [How Metabase executes SQL queries][how-metabase-executes-sql-queries]
+- [How Metabase executes SQL variables][how-metabase-executes-sql-variables]
+
+### I don’t know how to get the schema for my nested query
+
+**Subqueries** or **CTEs**
+
+You’ll need to manually run SELECT statements in a new SQL editor. Use the `LIMIT` clause to get a small sample of data.
+
+**Metabase saved question or model**
+
+Go to the saved question or model from the variables panel or by pasting the ID number into the Metabase search bar. Convert it to a SQL question and run the query with a `LIMIT` clause to get a small sample of data.
```

Comment:
You can limit the results in the query builder.

Reply:
I'm assuming that the SQL editor is being used in context of this troubleshooting topic :)